tests/base_test.py

```
@@ -0,0 +1,47 @@
+all test suites to ensure consistent test behavior and error reporting.
+"""
+class BaseTest:
+    """
+    Base class for all test suites.
+    Contains shared assertions and helpers - do not add domain logic here.
+    """
+    def assert_valid_base64(self, value: str) -> None:
+        """Verifies that a string is valid Base64."""
+        try:
+            base64.b64decode(value, validate=True)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)